src/index.ts

```diff
@@ -17,11 +17,27 @@ export const angularModule = angular.module(name, [])
 
         init: function($scope, el): void {
           if (!el.hasAttribute('popgun-listening')) {
-            el.addEventListener('PopgunContentSetup', function(e) {
-              let pop = popgun.getPopFromGroupId((<Element>e.target).getAttribute('popgun-group'));
-              $compile(pop.popOver.element)($scope);
-              $scope.$apply();
-            }, false);
+            el.addEventListener('PopgunContentSetup', (function($compileScope) {
+              return function (e) {
+                  let $newScope = $compileScope.$new();
```

Comment:
Yes, it is a possibility. I haven't particularly found a good solution other than thinking about exposing a conditional statement to state whether a new scope should be created or not (where default is true.)